craft_cli/messages.py

```diff
@@ -39,23 +41,51 @@ class _MessageInfo:
 # the different modes the Emitter can be set
 EmitterMode = enum.Enum("EmitterMode", "QUIET NORMAL VERBOSE TRACE")
 
+# the limit to how many log files to have
+LOG_FILES_LIMIT = 5
+
 
 def get_terminal_width() -> int:
     """Return the number of columns of the terminal."""
     return shutil.get_terminal_size().columns
 
 
-def get_log_filepath(appname):
-    """Provide a filepath for logging into."""
-    # XXX Facundo 2021-09-03: this will change heavily in next couple of branches
-    _, filepath = tempfile.mkstemp(prefix=f"{appname}-")
-    return filepath
+def get_log_filepath(appname: str) -> pathlib.Path:
+    """Provide a filepath for logging into.
+
+    The app name is used for both the directory where the logs are located and each log name.
+
+    Rules:
+    - use an appdirs provided directory
+    - base filename is <appname>.<timestamp with microseconds>.log
+    - it rotates until it gets to ROTATION_LIMIT
```

Comment:
ROTATION_LIMIT is probably the other constant
```suggestion
    - it rotates until it reaches :data:`.LOG_FILES_LIMIT`
```

Reply:
Improved. Don't know how to really see this, though.

Reply:
"make docs" at the root of the project should construct documentation

Reply:
It did, thanks.